### PR TITLE
[squid:ModifiersOrderCheck] Modifiers should be declared in the correct order

### PR DESCRIPTION
**Pull request by SosoTughushi**
This pull request is focused on resolving occurrences of Sonar rule 
squid:ModifiersOrderCheck - “Modifiers should be declared in the correct order”.

You can find more information about the issue here: 
https://dev.eclipse.org/sonar/rules/show/squid:ModifiersOrderCheck

Please let me know if you have any questions.
Soso Tughushi.

### CODE REVIEW

File: jdroid-android-google-inappbilling/src/main/java/com/android/vending/billing/IInAppBillingService.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 public interface IInAppBillingService extends android.os.IInterface {
 	
 	/** Local-side IPC implementation stub class. */
```
Comment (maxirosson) [inline jdroid-android-google-inappbilling/src/main/java/com/android/vending/billing/IInAppBillingService.java]: This class is auto generated. Please revert this change. Thanks

Comment (SosoTughushi) [inline jdroid-android-google-inappbilling/src/main/java/com/android/vending/billing/IInAppBillingService.java]: Reverted this change

File: jdroid-android-google-inappbilling/src/main/java/com/android/vending/billing/IInAppBillingService.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * is consumed. 4. An API to consume a purchase of an inapp item. All purchases of one-time in-app items are consumable
  * and thereafter can be purchased again. 5. An API to get current purchases of the user immediately. This will not
  * contain any consumed purchases.
- *
```
Comment (maxirosson) [inline jdroid-android-google-inappbilling/src/main/java/com/android/vending/billing/IInAppBillingService.java]: Please revert all the changes in this class. Thanks

**Comment by maxirosson** [comment]
Closing it because the collaborator didn't fix the pull request comments.
